Source/WebCore/page/ViewportConfiguration.h

```diff
@@ -145,6 +145,7 @@ class ViewportConfiguration {
     WEBCORE_EXPORT Parameters nativeWebpageParameters();
     static Parameters nativeWebpageParametersWithoutShrinkToFit();
     static Parameters nativeWebpageParametersWithShrinkToFit();
+    WEBCORE_EXPORT static Parameters pluginParameters();
```

Comment:
I was expecting the name to read more like `pluginDocumentParameters`

Reply:
I think that makes sense to me as well!